code/_onclick/hud/parallax.dm

```diff
@@ -75,11 +83,13 @@ var/list/parallax_on_clients = list()
 			C.screen -= bgobj
 		parallax_on_clients -= C
 		C.screen -= C.parallax_master
-		C.screen -= C.parallax_dustmaster
+		C.screen -= C.parallax_spacemaster
```

Comment:
But you return right after this - why would there be any masters on the screen at all?